src/schemathesis/extra/pytest_plugin.py

```diff
@@ -53,7 +55,15 @@ def _gen_items(self, endpoint: Endpoint) -> Generator[Function, None, None]:
         metafunc = self._parametrize(cls, definition, fixtureinfo)
 
         if not metafunc._calls:
-            yield create(SchemathesisFunction, name=name, parent=self.parent, callobj=funcobj, fixtureinfo=fixtureinfo)
+            yield create(
+                SchemathesisFunction,
+                name=name,
+                parent=self.parent,
+                callobj=funcobj,
+                fixtureinfo=fixtureinfo,
+                test_func=self.test_function,
+                originalname=self.name,
```

Comment:
This line might be done separately - it doesn't affect the current state (to reduce the size of this PR)